conda/environments/cudf_dev_cuda10.1.yml

```diff
@@ -40,6 +40,8 @@ dependencies:
   - flake8=3.8.3
   - black=19.10
   - isort=5.0.7
+  - mypy=0.782
+  - typing_extensions
   - pre_commit
```

Comment:
please specify a version for `typing_extensions` and `pre_commit`, if possible. (same goes for `...10.2.yml`, `...11.0.yml`, and `...meta.yml`

Reply:
There's tons of packages here for which we don't pin versions - I believe the reasoning is to allow as much flexibility for the conda dependency resolver. cc: @kkraus14 here to comment as I'm out of my element.

Reply:
Pinning a version of `pre_commit` is outside the scope of this PR. We could pin `typing_extensions` if desired, but I generally prefer to keep things open.